pom.xml

```
@@ -16,17 +18,14 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jenkins.version>2.346.1</jenkins.version>
         <groovy.version>2.4.21</groovy.version>
         <revision>1.35</revision>
         <changelist>-SNAPSHOT</changelist>
-        <!-- TODO: Move these three properties to the parent POM or use org.jenkins-ci:jenkins as the parent POM here -->
-        <incrementals-enforce-minimum.version>1.1</incrementals-enforce-minimum.version>
-        <incrementals-plugin.version>1.1</incrementals-plugin.version>
-        <incrementals.url>https://repo.jenkins-ci.org/incrementals/</incrementals.url>
         <scmTag>HEAD</scmTag>
     </properties>
 
-    <scm>
+    <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
         <connection>scm:git:git@github.com/cloudbees/groovy-cps.git</connection>
         <developerConnection>scm:git:ssh://git@github.com/cloudbees/groovy-cps.git</developerConnection>
```

SSH URLs are [deprecated](https://github.blog/2021-09-01-improving-git-protocol-security-github/). The following has been the most reliable replacement I have found:

```suggestion
        <developerConnection>scm:git:git@github.com:cloudbees/groovy-cps.git</developerConnection>
```

---

Thanks, modified in [e623ff9](https://github.com/cloudbees/groovy-cps/pull/120/commits/e623ff9584c3f46f36192c30ec5234d8e6e6215d).